cves/CVE-2010-3718.yml

```diff
@@ -43,7 +43,15 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  In Tomcat, web applications are normally allowed access to a particular work
+  directory. The applications are granted both read and write permissions so
+  they can use temporary files in that directory. Tomcat has an attribute that
+  dictates the location of that work directory. To prevent web applications from
+  changing the work directory, they should not be able to modify that attribute.
```

Comment:
Changing 'they' to something more specific may increase clarity.

Reply:
Thanks. I tried rewording that sentence and the description as a whole to make it a bit clearer.